runtime-modules/proposals/codex/src/benchmarking.rs

```diff
@@ -234,7 +234,7 @@ benchmarks! {
 
     create_proposal_funding_request {
         let i in 1 .. MAX_FUNDING_REQUEST_ACCOUNTS.try_into().unwrap();
-        let t in ...;
+//        let t in ...;
```

Comment:
Why did you comment such lines?

Reply:
> Looks good. However, this is a major question: how does it come that we have proposals to veto the bounty and don't have for creating one? Is there another PR for it?
I am not sure on those they were requested by the Pioneer team

> Also, I didn't understand why did you comment the title variables in benchmarks for some proposals. They could affect weights in benchmarking. Also, your local benchmarking runs discarded those variables in the autogenerated runtime file. Did you try to run it several times? It could flip back and forth - so, it's better to keep those parameters included.
Ok

Reply:
According to the designs, bounties are created with its own "create bounty" button, so we never thought that we need a proposal for that. I don't think it is needed to be fair.. 

Screenshot from https://pioneer-2-bounties.vercel.app/#/bounty/current

<img width="789" alt="Screenshot 2022-02-16 at 15 11 24" src="https://user-images.githubusercontent.com/93924525/154294497-d1efeb8a-936b-41ef-b5cd-9e62870667bb.png">

Reply:
and about the commented lines should be fixed in 12c507d7c0335e3a38dccb0c6d393bf8a96fda88

Reply:
@shamil-gadelshin @ignazio-bovo guys, I think I jumped the gun here with this comment and I may be wrong. 

Functionality-wise Councils can participate in bounties through proposals mechanism.

Council should be able to:

Create Bounty - via Create Bounty proposal that replicates Create Bounty flow (plus some proposal specific-steps)
Contribute Funding to Bounty
Withdraw Funding
And Switch Oracle https://github.com/Joystream/joystream/issues/2975 
(We don't need a Proposal to assign Oracle, since that is part of "Create Bounty" flow)
Veto Bounty

It does not seem to be logical to me for Work entries and work to get submitted via proposals, so this is probably safe to leave out.

Adding @bedeho here, as he has the best view on phasing out the Council participation in Bounties. From purely Pioneer perspective, that was not in my focus/ scope for release.